src/renderers/shared/utils/ReactErrorUtils.js

```diff
@@ -240,6 +244,17 @@ if (__DEV__) {
               'or switching to a modern browser. If you suspect that this is ' +
               'actually an issue with React, please file an issue.',
           );
+        } else if (isCrossOriginError) {
+          error = new Error(
+            "A cross-origin error was thrown. React doesn't have access to " +
+              'the actual error because it catches errors using a global ' +
+              'error handler, in order to preserve the "Pause on exceptions" ' +
+              'behavior of the DevTools. This is only an issue in DEV-mode; ' +
+              'in production, React uses a normal try-catch statement.\n\n' +
+              "It's recommended to serve JavaScript files from the same " +
+              'origin as your application.',
```

Comment:
Do you have to change your script to use the `crossorigin` attribute to `anonymous` for this to be respected by the browser though?

```html
<script src="https://example.com/example-framework.js"
        crossorigin="anonymous"></script>
```

I think that could be the issue.